cfgov/core/services.py

```diff
@@ -10,10 +12,17 @@
 from dateutil.parser import parse
 from v1.forms import CalenderPDFFilterForm
 
+
+class PDFReactorNotConfigured(Exception):
+    pass
+
 ## TODO: Update to python 3 when PDFreactor's python wrapper supports it.
 if six.PY2:
-    from core.lib.PDFreactor import *
-
+    try:
+        sys.path.append(os.environ.get('PDFREACTOR_LIB'))
```

Comment:
yea i was under that impression too, but tested it out locally and it works.

pull it out of the pdfreactor docs
```
import sys
sys.path.append("path/to/PDFreactor.py/")
from PDFreactor import *
```